package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "prestart": "tsc && yarn copy-graphql",
+    "setup-shows": "ts-node ./scripts/create-shows.ts",
```

Comment:
We're using one giant script to setup all data now (see #28), so no need for this line!